src/containers/app/index.js

```diff
@@ -111,34 +111,29 @@ class App extends React.Component {
     }
 
     render() {
-        const mainPath = (this.props.location.pathname.split('/').length > 0)
-            ? this.props.location.pathname.split('/')[1]
-            : '';
+        const mainPath =
+            this.props.location.pathname.split("/").length > 0 ? this.props.location.pathname.split("/")[1] : "";
 
         const showConnection =
-            ["account", "exchange", "loan", "reserves", "lock", "tryit", "loan"].indexOf(
-                mainPath
-            ) > -1;
+            ["account", "exchange", "loan", "reserves", "lock", "tryit", "loan"].indexOf(mainPath) > -1;
         return (
             <FeatureContext.Consumer>
                 {features => {
                     const showDash = features.dashboard && showConnection;
                     return (
                         <div className={showDash ? "Site Site__dash" : "Site"}>
                             <ScrollToTop />
-                            {showDash ? (
-                                <TopNav web3Connect={this.props.web3Connect} />
-                            ) : (
-                                    <AppMenu
-                                        web3Connect={this.props.web3Connect}
-                                        location={this.props.location}
-                                        showMenu={this.state.showMobileMenu}
-                                        toggleMenu={this.toggleMenu}
-                                    />
-                                )}
+                            <TopNav web3Connect={this.props.web3Connect} className={!showDash && "hide"} />
```

Comment:
imo instead of hiding the component by css you should not render it at all when `showDash` is false... replacing null in the line below would work

Reply:
If I using the other way I have a dependency issue and the browser back is not working (TopNavTitlePortal component search an element that is not rendered yet).